cli/server_request_command.go

```diff
@@ -138,6 +148,49 @@ func (c *SrvRequestCmd) healthz(_ *fisk.ParseContext) error {
 	return nil
 }
 
+func (c *SrvRequestCmd) profilez(_ *fisk.ParseContext) error {
+	nc, _, err := prepareHelper("", natsOpts()...)
+	if err != nil {
+		return err
+	}
+
+	opts := server.ProfilezEventOptions{
+		ProfilezOptions: server.ProfilezOptions{
+			Name:  c.profileName,
+			Debug: c.profileDebug,
+		},
+		EventFilterOptions: c.reqFilter(),
+	}
+
+	res, err := c.doReq("PROFILEZ", &opts, nc)
+	if err != nil {
+		return err
+	}
+
+	var response struct {
+		Resp server.ProfilezStatus `json:"data"`
+	}
+
+	var b []byte
+	for _, r := range res {
+		b = append(b, r...)
+	}
+	if err := json.Unmarshal(b, &response); err != nil {
+		return fmt.Errorf("no response")
+	}
+
+	if response.Resp.Error != "" {
+		return fmt.Errorf(response.Resp.Error)
+	}
+
+	// The response might be in binary format (some profiles can return
+	// gzipped responses) so write it to stdout as-is.
+	bio := bytes.NewBuffer(response.Resp.Profile)
+	bio.WriteTo(os.Stdout)
```

Comment:
We definitely cant write binary to stdout.

These req commands will fetch from multiple servers and the output would be unusable I think?

Maybe better to force a directory name be given and we output the results into files like `YYYMMDD-HHMMSS-<server id>-<profile name>` that way at least the data can be used and the utility is useful for running in a for loop to do ongoing profile gathering?

Reply:
Ah, didn't occur to me that multiple servers could respond. I will come up with something like you have suggested.

Reply:
We could also force a single server only on this one? Still binary to stdout is really bad UX